### PR TITLE
Added tlg5023.tlg010

### PR DESCRIPTION
**Pull request by planatheisa**
Checking Hook.

Also, I'm not quite sure about what to put under author. TLG gives "Varia" but "Scholia Graeca" is slightly less vague.

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     3    |
| `metadata_total`   |     4    |
| `nodes_count`      |   106    |
| `texts_passing`    |     2    |
| `texts_total`      |     2    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +5038 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg010/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | New      |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg010/__cts__.xml`                   | Failing  |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3372)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +4 |
| `metadata_total`   |       +4 |
| `nodes_count`      |     +106 |
| `texts_passing`    |       +2 |
| `texts_total`      |       +2 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +5038 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg010/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | New      |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg010/__cts__.xml`                   | Passing  |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3374)*

File: data/tlg5023/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg5023">
+  <ti:groupname xml:lang="lat">Scholia Graeca</ti:groupname>
```
Comment (sonofmun) [inline data/tlg5023/__cts__.xml]: The TLG has "Scholia in Euripidem" as the author. I think it makes sense to use that.

File: data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,636 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="lat">Vitae Euripidis</title>
+                <editor>Karl Wilhelm Dindorf</editor>
+                <author xml:lang="lat">Scholia Graeca</author>
```
Comment (sonofmun) [inline data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml]: As above, I would use "Scholia in Euripidem" as the author.

File: data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,636 @@
...
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Scholia Graeca in Euripidis tragoedias</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Gulielmus Dindorfius</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg5023" xml:lang="lat">Scholia Graeca</author>
```
Comment (sonofmun) [inline data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml]: Use "Scholia in Euripidem"

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +5 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +1571 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg010/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | New      |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg010/__cts__.xml`                   | Passing  |
| `./data/tlg5023/tlg010/tlg5023.tlg010.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3424)*

**Pull request merged**